### PR TITLE
Add hostNetwork in operator deployment needed when using Cilium CNI

### PR DESCRIPTION
**Pull request by samba1112**
… using Cilium CNI

Signed-off-by: Sambashiva <sambasiva1112@gmail.com>

_Provide a description of what has been changed_

### Checklist

- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [ ] A PR is opened to update KEDA core ([repo](https://github.com/kedacore/keda)) *(if applicable, ie. when deployment manifests are modified)*
- [ ] README is updated with new configuration values *(if applicable)*

Fixes #

### CODE REVIEW

**Comment by samba1112** [comment]
This is reg this Bug 
https://github.com/kedacore/keda/issues/3778

File: keda/values.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ imagePullSecrets: []
 
 operator:
   name: keda-operator
+  useHostNetwork: false
```
Comment (JorTurFer) [inline keda/values.yaml:25]: Should we define ` dnsPolicy` too as we do in the metrics server?
https://github.com/kedacore/charts/blob/main/keda/values.yaml#L40-L42

It seems that it's required to change it if we use `hostNetwork`

Comment (samba1112) [reply keda/values.yaml:25]: By default when i install helm chart `dnsPolicy: ClusterFirst` got assigned

**Comment by JorTurFer** [comment]
As we have commented by slack, this PR isn't necessary

**Pull request closed without merging**